playwright-e2e/tests/dsm/miscellaneous/permissions/view-seq-order-permissions.spec.ts

```diff
@@ -0,0 +1,102 @@
+                  UserPermission.SEQUENCING_ORDER_VIEW_STATUS
+                ]);
+            })
+
+            await test.step('Verify that the current DSM user is able to view the Clinical Order page (via the Samples menu)', async () => {
+              await navigation.selectFromSamples(SamplesNavEnum.CLINICAL_ORDERS);
+              const clinicalOrdersPage = new ClinicalOrdersPage(page);
+              await clinicalOrdersPage.waitForReady();
+          })
+
+            await test.step('Verify that the current DSM user is able to view the Clinical Order Columns in Participant List', async () => {
+              const participantListPage = await navigation.selectFromStudy<ParticipantListPage>(StudyNavEnum.PARTICIPANT_LIST);
+              await participantListPage.assertPageTitle();
+              await participantListPage.waitForReady();
+
+              const customizeViewPanel = participantListPage.filters.customizeViewPanel;
+              await customizeViewPanel.open();
+              await customizeViewPanel.isDisplayed(['Clinical Orders Columns']);
+
+              //Filter the list to display study participants with a Not-Empty Clinical Orders Columns -> Clinical Order ID column
+              await customizeViewPanel.selectColumns('Clinical Orders Columns', ['Clinical Order Id']);
+
+              const searchPanel = participantListPage.filters.searchPanel;
+              await searchPanel.open();
+              await searchPanel.checkboxes('Status', { checkboxValues: ['Enrolled'] });
+              await searchPanel.text('Clinical Order Id', { additionalFilters: [AdditionalFilter.NOT_EMPTY]});
+              await searchPanel.search();
+
+              const participantListTable = participantListPage.participantListTable;
+
+              //Check that at least 1 study participant already has a clinical order id
+              const numberOfParticipants = await participantListTable.numOfParticipants();
+              expect(numberOfParticipants).toBeGreaterThan(0);
+              await participantListTable.openParticipantPageAt(0);
+
+              const participantPage = new ParticipantPage(page);
+              await participantPage.assertPageTitle();
+              await participantPage.clickTab(TabEnum.SEQUENCING_ORDER);
+
+              const sequencingOrderTab = new SequencingOrderTab(page);
+              await sequencingOrderTab.waitForReady();
+              //Verify that the current DSM user is not able to submit sequencing orders for the chosen study participant
+              await sequencingOrderTab.assertNoSamplesCanBeSelected();
+              await sequencingOrderTab.assertPlaceOrderButtonNotVisible();
+              await sequencingOrderTab.fillAvailableCollectionDateFields({canPlaceClinicalOrder: false});
+            })
+
+            await test.step('Verify that the current DSM user is able to input collection dates for samples that do not have them', async () => {
+                //Stuff here
```

Comment:
What "stuff" should go here?

Reply:
it's taken cared of in the above step - inputting collection dates - this is a holdover from planning out test